source/industry-solutions/media-personalization.txt

```diff
@@ -0,0 +1,190 @@
+.. _arch-center-is-ai-media-personalization:
+
+==========================================================
+AI-driven Media Personalization: MongoDB and Vector Search
+==========================================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
```

Comment:
```suggestion
   :values: tutorial
   
.. meta:: 
   :keywords: GenAI, Atlas
   :description: Discover how to leverage MongoDB and Vector Search effectively to create more engaging, personalized user experiences.
```
IS articles are generally tutorials that show how to do something specific, rather than references that explain how things work. I also copied in a similar `meta` block to the one I used in my article.